website/src/pages/pricing.tsx

```diff
@@ -48,120 +72,152 @@ export default ((props: any) => (
+                                                value={true}
+                                                tag="li"
+                                                className="list-group-item bg-transparent border-0 px-0"
+                                            />
+                                            <PricingPlanFeature
+                                                info={{ label: "Email support", description: "" }}
+                                                value={true}
+                                                tag="li"
+                                                className="list-group-item bg-transparent border-0 px-0"
+                                            />
+                                        </ol>
+                                    </div>
+                                    <div className="col-md-6 mx-auto mb-4 pl-6">
+                                        <a
+                                            className={`pricing-plan__button btn btn-outline-primary w-100 mx-auto my-0`}
+                                            href="https://sourcegraph.com/subscriptions/new"
+                                        >
+                                            Create a team
+                                        </a>
+                                    </div>
+                                </div>
+
+                            </div>
+                        </div>
+                    </div>
+
                     <div className="py-3 text-center">
                         <h3>Education and nonprofit discounts</h3>
                         <p>
-                            Sourcegraph supports the work of educational organizations and nonprofits. Please{' '}
+                            Sourcegraph supports the work of educational organizations and nonprofits.<br />
+                            Please{' '}
                             <a href="/contact/sales/?form_submission_source=pricing-nonprofit">contact us</a> about
                             discounts for your development teams.
                         </p>
                     </div>
+
                     <hr className="my-4" />
+
+                    <div className="row justify-content-center pt-md-4">
+                        <div className="col-lg-10 text-center">
+                            <blockquote className="blockquote pricing-page__quote pricing-page__quote--in-content">
+                                <p>
+                                    Sourcegraph pays for itself many times over—it’s a game changer.
+                                </p>
+                                <footer className="blockquote-footer">François Jehl, Senior Engineering Manager, Criteo</footer>
+                                <div className="d-flex justify-content-center my-4">
+                                    <img src="/external-logos/criteo-logo.svg" width="156px" alt="Criteo" />
```

Comment:
@jeanduplessis - can this logo link here https://about.sourcegraph.com/case-studies/criteo-tackles-big-code/